config/locales/client.en.yml

```diff
@@ -173,6 +178,7 @@ en:
           je: "Jersey"
           jp: "Japan"
           kr: "Korea (Republic of)"
+          kz: "Kazakhstan (Republic of)"
```

Comment:
There were missing translations for this one^ and the two Brazil ones above:

![missing-translations](https://user-images.githubusercontent.com/103613060/170415890-1f551053-338b-463c-a065-fd41400aef96.png)